src/utils/utils.cpp

```diff
@@ -409,6 +597,66 @@ void EchoInfo(const std::string &message) {
   }
 }
 
+void EchoStats(const std::map<std::string, std::int64_t> &stats) {
+  for (const auto &[key, value] : stats) {
+    if (value == 0) {
+      continue;
+    }
+    if (key == "nodes-created") {
+      std::cout << value << " nodes have been created." << std::endl;
+    }
+    if (key == "nodes-deleted") {
+      std::cout << value << " nodes have been deleted." << std::endl;
+    }
+    if (key == "relationships-created") {
+      std::cout << value << " relationships have been created." << std::endl;
+    }
+    if (key == "relationships-deleted") {
+      std::cout << value << " relationships have been deleted." << std::endl;
+    }
+    if (key == "labels-added") {
+      std::cout << value << " labels have been created." << std::endl;
+    }
+    if (key == "labels-removed") {
+      std::cout << value << " labels have been deleted." << std::endl;
+    }
+    if (key == "properties-set") {
+      std::cout << value << " properties have been updated." << std::endl;
+    }
```

Comment:
these can be `else if` or early return, key cannot have more than one value at the same time.

Reply:
You can have multiple different changes in the same query, e.g. creating multiple nodes and edges

Reply:
but inside the for loop key represents a single value inside the notification map. Also, what value can be equal to multiple different values at the same time :smile:

Reply:
sry didn't see the loop in the previewed changes, you are right